cluster-scope/base/external-secrets.io/externalsecrets/open-cluster-management-observability-alertmanager-config/externalsecret.yaml

```diff
@@ -92,7 +92,7 @@ spec:
               - receiver: slack-notifications-prod-rhods-ope
                 group_by: ['...']
                 group_wait: 0s
-                repeat_interval: 0s
+                # repeat_interval cannot be zero, group_interval cannot be zero
```

Comment:
what does it default to if it's not set here at all?

Reply:
it is 4 hours. https://github.com/prometheus/alertmanager/issues/860#issuecomment-467585781
and I need to get it down to less for my specific case.
but right now, alert manager pod is not starting at all, and I want at least the other alerst running as usual.